docker-compose.yml

```diff
@@ -86,36 +79,5 @@
     "env_file": ".env",
     "stdin_open": true,
     "tty": true
-  },
-  "ipython": {
-    "build": "ipython",
-    "ports": [
-      "8888"
-    ],
-    "links": [
-      "bigmetadata_daemon",
-      "postgres"
-    ],
-    "volumes": [
-      ".:/bigmetadata"
-    ],
-    "environment": {
-      "PYTHONPATH": "/bigmetadata",
-      "PGHOST": "postgres",
-      "PGUSER": "docker",
-      "PGPASSWORD": "docker",
-      "PGDATABASE": "test",
-      "LC_ALL": "C.UTF-8",
-      "LANG": "C.UTF-8"
-    },
-    "env_file": ".env",
-    "working_dir": "/tmp/ipython",
-    "command": "sh -c /bigmetadata/ipython/start.sh"
-  },
-  "redis": {
```

Comment:
Why do we have a redis image?

Reply:
No idea. It was linked to the luigi daemon, and the daemon supports using redis as a backend. However, the current configuration uses psql as the backend, so I guess it's just legacy.